live-build/base/config/package-lists/minimal.list.chroot

```diff
@@ -43,3 +43,4 @@ open-vm-tools
 openssh-server
 ubuntu-minimal
 auditd
+python2.7
```

Comment:
I don't feel strongly about this, but I'm mildly concerned about adding Python 2.7 to all images given that [Python 2.7 will not be maintained in 1 year and 6 months from now](https://pythonclock.org/). If the Python 2.7 package is installed in all variants, it will be easier for people to start depending on it by accident, which will make it more difficult to rip out these dependencies in a release or two when Python 2.7 is EOL'd. I have a slight preference for limiting `python2.7` to the DCenter variant (if possible), though I don't feel strongly about this.

Reply:
In general I think I agree.. but.. I asked George to explicitly add it here, since it was being pulled in already (likely from a dependency of one of these other packages, but I haven't verified which), and George was already using it without really knowing how it was getting pulled in.

So, if we really don't want to add python2.7 to the base configuration, we need to figure out which other package in the base configuration depends on it, and remove that package(s) too.

Reply:
FYI, these are the packages we use in the standard image that currently depend on Python 2.7:

```
bpfcc-tools
cppcheck
dstat
fio
python-bpfcc
python-netaddr
python-rtslib-fb
zfs-test
zfsutils-linux
```

We should probably work on getting the ZFS stuff to stop depending on Python 2.7.

I think I still have a slight preference for keeping the explicit installation of Python 2.7 to the `dcenter` variant. That way, once the above utilities stop depending on Python 2.7, we'll stop pulling it in (but will keep doing so for the `dcenter` variant, as long as it needs Python 2.7).

Reply:
That reasonable to me; I'm OK moving the explicit python2.7 dependency into the dcenter Ansible role.

The catch being, the dcenter ansible playbook then should use python3 as the python interpreter (like the other playbooks), since we can't assume python2 will be there prior to running the role. That'll require other changes, e.g. to how "pip" is used in that role; but if we can just stop using pip in favor of the python packages (I think we should, if we can), this might not be a problem.